src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -290,20 +291,7 @@ private void handleInvalidFileError(
             .getFor(cause.getClass())
             .ifPresent(errorCode -> {
                 try {
-                    this.notificationsQueueHelper.sendMessage(
-                        new ErrorMsg(
-                            UUID.randomUUID().toString(),
-                            eventId,
-                            zipFilename,
-                            containerName,
-                            null,
-                            null,
-                            errorCode,
-                            cause.getMessage()
-                        )
-                    );
-
-                    log.info("Sent error notification for file {} in container {}", zipFilename, containerName);
+                    sendErrorMessageToQueue(zipFilename, containerName, eventId, errorCode, cause);
```

Comment:
if you really want to extrac this to a function, perhaps include the catch block (so that it's self contained)?

Reply:
This method will still stay as it is - it's long enough. I can wrap it in `trySendErrorMessageToQueue`, which would include exception handling.